student-information-management-application/src/main/java/com/student/information/management/add/data/student/dao/StudentDao.java

```diff
@@ -0,0 +1,15 @@
+package com.student.information.management.add.data.student.dao;
+
+import com.student.information.management.add.app.model.student.Student;
+
+import java.sql.ResultSet;
+import java.util.List;
+/**
+ * This is the Student Dao Interface.
+ */
+public interface StudentDao {
+    List<Student> getAllStudents();
+    Student getStudentById(String student_id) throws RuntimeException;
```

Comment:
why is an interface throwing an exception?

Reply:
All Done po.